config/kubernetes/org.yaml

```diff
@@ -892,6 +894,7 @@ members:
 - SergeyKanzhelev
 - sethmccombs
 - sethpollack
+- sfotony
 - sflxn
```

Comment:
ordering fix:
```suggestion
- sflxn
- sfotony
```

Reply:
Thanks for the catch @palnabarun. Fix coming up right away!

Reply:
Fixed it :)